stock_product_pack/models/product_product.py

```diff
@@ -48,6 +48,23 @@ def _compute_quantities_dict(
             }
         return res
 
+    @api.depends(
+        "stock_move_ids.product_qty",
+        "pack_line_ids.product_id.stock_move_ids.product_qty",
+        "stock_move_ids.state",
+        "pack_line_ids.product_id.stock_move_ids.state",
+        "stock_move_ids.quantity",
+        "pack_line_ids.product_id.stock_move_ids.quantity",
+    )
+    @api.depends_context(
+        "lot_id",
```

Comment:
You don't need to add fields that are in base module. ORM will concatenate them

Reply:
Done. Could you please help to review it again?

Reply:
ping @rousseldenis